docs/getting-started.md

```diff
@@ -90,24 +92,24 @@ example-python: Build ID: 01BSY5R8J45QHG9D3B17PAXMGN
 Now that the application has been deployed, we can connect to our app.
 
 ```shell
-$ draft connect
-Connect to python:8080 on localhost:39551
+$ draft connect -p 8081:8080
+Connect to python:8080 on localhost:8081
 172.17.0.1 - - [13/Sep/2017 19:10:09] "GET / HTTP/1.1" 200 -
 ```
 
-Note that it may take some time for the app to deploy, so if you see a message like "Error: could not find a ready pod", just wait a little longer for the image to be deployed.
-
-`draft connect` works by creating a local environment for you to test your app. It proxies a connection from the pod running in minikube to a localhost url that you can use to see your application working. It will also print out logs from your application.
+`draft connect` is the command used to interact with the application deployed on your cluster. It works by creating proxy connections to the ports exposed by the containers in your pod, while also streaming the logs from all containers.
 
 In another terminal window, we can connect to our app using the address displayed from `draft connect`'s output.
 
 ```shell
-$ curl localhost:55139
+$ curl localhost:8081
 Hello, World!
 ```
 
 Once you're done playing with this app, cancel out of the `draft connect` session using CTRL+C.
 
+> Note that you can use the flag `draft up --auto-connect` in order to have the application automatically connect once the deployment is done, and you can customize the local ports for the application either using the `-p` flag or through the `override-ports` field in `draft.toml`. More info in [dep-007.md][dep007]
+
```

Comment:
This made me think that I could specify the ports on the `draft up` command since it was all in the same sentence. I think it might be more clear if we first specify that we can customize ports using the `-p` flag and override-ports field and then mention in a separate sentence that we can also use ` draft up --auto-connect` for convenience for the same behavior. Let me know if that makes sense.